setup.py

```diff
@@ -36,6 +38,7 @@
             "coveralls>=3.3.1",
             "types-setuptools",
             "types-requests",
+            "networkx",
```

Comment:
Can pin versions here for `black`, `networkx`, `types-setuptools`, and `types-requests`

Reply:
sure will do

Reply:
addressed in [ab082a5](https://github.com/arangoml/pyg-adapter/pull/2/commits/ab082a5921e78b1e9e6e0c679bb9f1f6210bf418)